cms/templates/container.html

```diff
@@ -12,7 +12,7 @@
 from django.utils.translation import ugettext as _
 from openedx.core.lib.js_utils import escape_json_dumps
 %>
-<%block name="title">${xblock.display_name_with_default} ${xblock_type_display_name(xblock) | h}</%block>
+<%block name="title">${xblock.display_name_unescaped} ${xblock_type_display_name(xblock) | h}</%block>
```

Comment:
@robrap, thanks for your detailed review.

I have fixed the title, in line with your second recommendation: "*switch to some form of the following*".